### PR TITLE
Tutorials navigation taxonomy updated

### PR DESCRIPTION
**Pull request by bpurkaystha**
### Description

https://app.asana.com/0/1201992122798724/1202220412576182/f

- Navigation taxonomy update

### CODE REVIEW

File: _includes/nav.html [hunk 1]
```diff
@@ -65,20 +64,12 @@
...
-              <a href="https://developer.confluent.io/learn/kafka-transactions-and-guarantees/">Transactions &amp;
-                Guarantees in Kafka</a>
-              <a href="https://developer.confluent.io/learn/kafka-storage-and-processing/">Processing &amp; Storage
-                Fundamentals of Kafka</a>
-              <a href="https://developer.confluent.io/learn/kafka-performance/">Kafka Performance</a>
-              <a href="https://developer.confluent.io/learn/cloud-native-kafka/">Cloud-native Kafka</a>
-              <a href="https://developer.confluent.io/learn/streaming-database-systems/">Streaming Database Systems</a>
-              <a href="https://developer.confluent.io/learn/testing-kafka/">Testing Apache Kafka</a>
-              <a href="https://developer.confluent.io/#kafka-internals">Explore Kafka's Internals</a>
+              <a href="https://developer.confluent.io/learn">Articles</a>
```
Comment (iPzard) [inline _includes/nav.html:67]:
```suggestion
              <a href="https://developer.confluent.io/learn/">Articles</a>
```

File: _includes/nav.html [hunk 2]
```diff
@@ -65,20 +64,12 @@
...
+              <a href="https://developer.confluent.io/patterns/">Patterns</a>
+              <a href="https://developer.confluent.io/learn/apache-kafka-faqs/">FAQs</a>
+              <a href="https://developer.confluent.io/100-days-of-code">100 Days of Code</a>
```
Comment (iPzard) [inline _includes/nav.html:70]:
```suggestion
              <a href="https://developer.confluent.io/100-days-of-code/">100 Days of Code</a>
```

File: _includes/nav.html [hunk 3]
```diff
@@ -87,20 +78,14 @@
...
-              <a href="https://developer.confluent.io/tutorials/">Stream Processing Cookbook</a>
-              <a href="https://developer.confluent.io/?build=apps#designbuildrun">Developers: Build Apps</a>
-              <a href="https://developer.confluent.io/?build=pipelines#designbuildrun">Data Engineers: Build Pipelines
-                and Integrate</a>
-              <a href="https://developer.confluent.io/?build=operate#designbuildrun">Admins: Operate</a>
-              <a href="https://developer.confluent.io/demos-examples/">Demos &amp; In-depth Examples</a>
-              <a href="https://developer.confluent.io/100-days-of-code/">100 Days Of Code</a>
+              <a href="https://developer.confluent.io/get-started/">Language Guides</a>
+              <a href="https://developer.confluent.io/tutorials/">Tutorials</a>
+              <a href="https://developer.confluent.io/tutorials/use-cases.html">Stream Processing Cookbook</a>
```
Comment (iPzard) [inline _includes/nav.html:87]:
```suggestion
              <a href="https://developer.confluent.io/tutorials/">Stream Processing Cookbook</a>
```

File: _includes/nav.html [hunk 4]
```diff
@@ -112,16 +97,12 @@
...
-                Events</a>
-              <a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/blog/">Blog</a>
-              <a href="https://developer.confluent.io/podcast/">Podcast</a>
-              <a target="_blank" rel="noopener noreferrer"
-                href="https://www.confluent.io/community/ask-the-community/">Ask the Community</a>
-              <a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/nominate/">Community
-                Catalysts</a>
+            <div class="accordion-panel">
+              <a target="_blank" rel="noopener noreferrer" href="https://www.kafka-summit.org/">Kafka Summit Conference</a>
+              <a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/community/">Meetups &amp;Events</a>
```
Comment (iPzard) [inline _includes/nav.html:102]:
```suggestion
              <a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/community/">Meetups &amp; Events</a>
```

File: _includes/nav.html [hunk 5]
```diff
@@ -112,16 +97,12 @@
...
+              <a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/community/ask-the-community/">Ask the Community</a>
+              <a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/nominate/">Community Catalysts</a>
+              <a rel="noopener noreferrer" href="#" style="font-weight: normal">Catalyst spotlight</a>
```
Comment (iPzard) [inline _includes/nav.html:105]: Why is there a link to nothing?

File: _includes/nav.html [hunk 6]
```diff
@@ -166,45 +149,31 @@
...
-                Guarantees in Kafka</a></li>
-            <li><a href="https://developer.confluent.io/learn/kafka-storage-and-processing/">Processing &amp; Storage
-                Fundamentals of Kafka</a></li>
-            <li><a href="https://developer.confluent.io/learn/kafka-performance/">Kafka Performance</a></li>
-            <li><a href="https://developer.confluent.io/learn/cloud-native-kafka/">Cloud-native Kafka</a></li>
-            <li><a href="https://developer.confluent.io/learn/streaming-database-systems/">Streaming Database
-                Systems</a></li>
-            <li><a href="https://developer.confluent.io/learn/testing-kafka/">Testing Apache Kafka</a></li>
-            <li><a href="https://developer.confluent.io/#kafka-internals">Explore Kafka's Internals</a></li>
+            <li><a href="https://developer.confluent.io/learn">Articles</a></li>
```
Comment (iPzard) [inline _includes/nav.html:160]:
```suggestion
            <li><a href="https://developer.confluent.io/learn/">Articles</a></li>
```

File: _includes/nav.html [hunk 7]
```diff
@@ -166,45 +149,31 @@
...
+            <li><a href="https://developer.confluent.io/patterns/">Patterns</a></li>
+            <li><a href="https://developer.confluent.io/learn/apache-kafka-faqs/">FAQs</a></li>
+            <li><a href="https://developer.confluent.io/100-days-of-code">100 Days of Code</a></li>
```
Comment (iPzard) [inline _includes/nav.html:163]:
```suggestion
            <li><a href="https://developer.confluent.io/100-days-of-code/">100 Days of Code</a></li>
```

File: _includes/nav.html [hunk 8]
```diff
@@ -216,12 +185,11 @@
                 Conference</a></li>
             <li><a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/community/">Meetups &amp;
                 Events</a></li>
-            <li><a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/blog/">Blog</a></li>
-            <li><a href="https://developer.confluent.io/podcast/">Podcast</a></li>
             <li><a target="_blank" rel="noopener noreferrer"
                 href="https://www.confluent.io/community/ask-the-community/">Ask the Community</a></li>
             <li><a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/nominate/">Community
                 Catalysts</a></li>
+            <li><a rel="noopener noreferrer" href="#" style="font-weight: normal">Catalyst spotlight</a></li>
```
Comment (iPzard) [inline _includes/nav.html:192]: Same here.. the `href` is just set to a hash.

File: _includes/nav.html (hunk 5 above)
Comment (bpurkaystha) [reply _includes/nav.html:105]: That's how it's there in the contentful.

File: _includes/nav.html [hunk 9]
```diff
@@ -233,6 +201,9 @@
             <li>
               <a href="https://docs.confluent.io/home/overview.html" rel=" noreferrer" data-test-id="link">Docs Home</a>
             </li>
+            <li>
+              <a href="https://docs.confluent.io/cloud/current/overview.html" rel=" noreferrer" data-test-id="link">Confluent Cloud</a>
```
Comment (iPzard) [inline _includes/nav.html:205]:
```suggestion
              <a href="https://docs.confluent.io/cloud/current/overview.html" rel="noopener noreferrer" data-test-id="link">Confluent Cloud</a>
```

File: _includes/nav.html (hunk 8 above)
Comment (bpurkaystha) [reply _includes/nav.html:192]: Yes @iPzard , this is how it's there in the contentful.

File: _includes/nav.html [hunk 10]
```diff
@@ -166,45 +149,31 @@
...
-            <li><a href="https://developer.confluent.io/tutorials/">Stream Processing Cookbook</a></li>
-            <li><a href="https://developer.confluent.io/?build=apps#designbuildrun">Developers: Build Apps</a></li>
-            <li><a href="https://developer.confluent.io/?build=pipelines#designbuildrun">Data Engineers: Build Pipelines
-                and Integrate</a></li>
-            <li><a href="https://developer.confluent.io/?build=operate#designbuildrun">Admins: Operate</a></li>
-            <li><a href="https://developer.confluent.io/demos-examples/">Demos &amp; In-depth Examples</a></li>
-            <li><a href="https://developer.confluent.io/100-days-of-code/">100 Days Of Code</a></li>
+            <li><a href="https://developer.confluent.io/get-started/">Language Guides</a></li>
+            <li><a href="https://developer.confluent.io/tutorials/">Tutorials</a></li>
+            <li><a href="https://developer.confluent.io/tutorials/use-cases.html">Stream Processing Cookbook</a></li>
```
[the remark this answers is not in the document]
Comment (bpurkaystha) [reply _includes/nav.html]: Updated.

File: _includes/nav.html (hunk 7 above)
Comment (bpurkaystha) [reply _includes/nav.html]: Updated.

File: _includes/nav.html (hunk 6 above)
Comment (bpurkaystha) [reply _includes/nav.html]: Updated.

File: _includes/nav.html (hunk 3 above)
Comment (bpurkaystha) [reply _includes/nav.html]: Upadted.

File: _includes/nav.html (hunk 1 above)
Comment (bpurkaystha) [reply _includes/nav.html]: Updated.

File: _includes/nav.html (hunk 8 above)
Comment (hartantothio) [reply _includes/nav.html:192]: Does this do anything? Why only hash?

Comment (bishwa420) [reply _includes/nav.html]: After talking to @rmoff , this sub-menu has been removed. Thanks.

File: _includes/nav.html (hunk 5 above)
Comment (bishwa420) [reply _includes/nav.html]: Removed now.

File: _includes/nav.html [hunk 11]
```diff
@@ -65,20 +64,13 @@
...
-              <a href="https://developer.confluent.io/learn/streaming-database-systems/">Streaming Database Systems</a>
-              <a href="https://developer.confluent.io/learn/testing-kafka/">Testing Apache Kafka</a>
-              <a href="https://developer.confluent.io/#kafka-internals">Explore Kafka's Internals</a>
+              <a href="https://developer.confluent.io/learn/">Articles</a>
+              <a href="https://developer.confluent.io/patterns/">Patterns</a>
+              <a href="https://developer.confluent.io/learn/apache-kafka-faqs/">FAQs</a>
+              <a href="https://developer.confluent.io/100-days-of-code/">100 Days of Code</a>
+              <a href="https://www.confluent.io/blog/">Blog</a>
+              <a href="https://developer.confluent.io/podcast/">Podcast</a>
+              <a href="https://developer.confluent.io/live/">Confluent Developer Live</a>
```
Comment (bbejeck) [inline _includes/nav.html:73]: I don't see this link from the `Learn` menu on the KT site

**Review by bbejeck — COMMENTED** [review]
Overall LGTM minus one comment I have

File: _includes/nav.html (hunk 11 above)
Comment (bishwa420) [reply _includes/nav.html:73]: You mean on the production? It's a new menu item being added here.

**Review by bbejeck — APPROVED** [review]
LGTM

**Pull request merged**